docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,539 @@
+        /* Error handling */
+
+    ret = noti_ex_item_entry_set_text(entry_item, "entry_text");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Progress Item:
+
+  This item allows you to create a progress bar and update the progress data such as percentage or time.
+    ```
+    noti_ex_item_h progress_item = NULL;
+
+    ret = noti_ex_item_progress_create(&progress_item, "progress_id", 0, 50, 100);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+
+    ret = noti_ex_item_progress_set_current(progress_item, 70);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Time Item:
+
+  This item contains time information.
+    ```
+    ret = noti_ex_item_time_create(&time_item, "time_id", time_info);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+<a name="group"></a>
+## Creating Group Item and Adding Child
+
+- You can add an item as a child of a group using `noti_ex_item_group_add_child()`:
+
+  A group item is a series of items. You can add items to a group.
+  If you add multiple items to the child, the items are displayed in the order that they are added. There can be multiple groups in a notification.
+    ```
+    noti_ex_item_h group_item = NULL;
+    noti_ex_item_h child_item = NULL;
+
+    ret = noti_ex_item_group_create(&group_item, "group_id");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+
+    ret = noti_ex_item_button_create(&child_item, "button_id", "button_title");
+    ret = noti_ex_item_group_add_child(group_item, child_item);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- You can use `noti_ex_item_set_direction()` to decide whether to display the child vertically or horizontally:
+    ```
+    ret = noti_ex_item_group_set_direction(group_item, true);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+<a name="set"></a>
+## Setting Notification-ex Attributes
+
+You can set the following attributes for a notification:
+
+- ID:
+
+  All items are identified by an ID attribute. By default, ID is specified while creating an item but it can also be set separately.
+    ```
+    ret = noti_ex_item_set_id(item_handle, "new_id");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Style:
+
+  The style attribute sets the color, padding, and geometry for an item. You can also specify the background image and color.
+
+  - Color
+
+    Color information can be defined as a `notification_ex color` handle with alpha, red, green, and blue value.
+    ```
+    noti_ex_color_create(noti_ex_color_h *handle, unsigned char a, unsigned char r, unsigned char g, unsigned char b);
+    ```
+
+  - Padding
+
+    Padding information can be defined as a `notification_ex padding` handle with left, right, top, and bottom values.
+    ```
+    noti_ex_padding_create(noti_ex_padding_h *handle, int left, int top, int right, int bottom);
```

Comment:
minor issue. The order is different.

In description :  ...left, right, top, and bottom values.
In sample code : ...int left, int top, int right, int bottom);